### ISSUE TITLE
SyntaxError: Unexpected closing tag "script". 

### ISSUE THREAD

**Issue by JusticHentai** [johnsoncodehk/volar#1670]
`@volar/vue-typescript": "0.39.4"`
`prettier-plugin-organize-imports": "3.0.1"`

code:
```
<template>
  <div class="douza-app">
    <Home />
  </div>
</template>

<script lang="ts">
import Home from '@pages/h5/views/Home.vue'
import { Component, Vue } from 'vue-property-decorator'

@Component({
  name: 'app',
  components: {
    Home,
  },
})
export default class app extends Vue {
  created() {
       console.log('test')
  }

  mounted() {}
}
</script>

<style lang="scss">
@include h5-config;
</style>
```

run
```
prettier --write .
```

error
```
packages\templates\code-mode\src\pages\h5\App.vue
[error] packages\templates\code-mode\src\pages\h5\App.vue: SyntaxError: Unexpected closing tag "script". It may happen when the tag has already been closed by another tag. For more info see https://www.w3.org/TR/html5/syntax.html#closing-elements-that-have-implied-end-tags (20:1)
[error]   18 |   mounted() {}
[error]   19 | }
[error] > 20 | </script>
[error]      | ^^^^^^^^^
[error]   21 |
[error]   22 | <style lang="scss">
[error]   23 | @include h5-config;
```

**Comment by Santobert** [comment]
This issue seems to apply only to windows machines. Some more debugging info can be found here: https://github.com/simonhaenisch/prettier-plugin-organize-imports/issues/66

**Comment by JusticHentai** [comment]
> This issue seems to apply only to windows machines. Some more debugging info as well as a minimal example can be found here: [simonhaenisch/prettier-plugin-organize-imports#66](https://github.com/simonhaenisch/prettier-plugin-organize-imports/issues/66)

in window you can try to use it in this version

`"@volar/vue-typescript": "0.38.9"`
`"prettier-plugin-organize-imports": "3.0.0"`

or use the same feature in `eslint-plugin-import`

**Comment by johnsoncodehk** [comment]
I think `prettier-plugin-organize-imports` using `\\` style path to `vue-typescript` and `typescript`, but it's invalid for TypeScript language service.

@simonhaenisch My suggestion is refactor the code with https://github.com/johnsoncodehk/typesafe-path, I can try open a PR when I have time if you want.

**Comment by simonhaenisch** [comment]
I recently did this fix: https://github.com/simonhaenisch/prettier-plugin-organize-imports/commit/91377de958aa7beb29c1d17483d0f23630a3335d

It was released in [v3.1.1](https://github.com/simonhaenisch/prettier-plugin-organize-imports/releases/tag/v3.1.1) so by upgrading the plugin the issue should be fixed, I guess?
